src/jitlayers.cpp

```diff
@@ -308,6 +308,27 @@ void NotifyDebugger(jit_code_entry *JITCodeEntry)
 }
 // ------------------------ END OF TEMPORARY COPY FROM LLVM -----------------
 
+// Resolve compiler-rt functions in the shared library that we created from compiler-rt
+static uint64_t resolve_compiler_rt(const char *name)
+{
+#if defined(__APPLE__)
+    static const char *const compiler_rt_lib = "@rpath/julia/libcompiler-rt";
```

Comment:
@vtjnash Is this the correct form for a rpath library? Or do I need to add `.dylib`?